modules/project-docs/examples/MigratingSDKCodeTo3n.java

```diff
@@ -48,19 +48,25 @@ private void init() {
     collection = bucket.defaultCollection();
   }
 
-  public void migrating_sdk_code_to_3_n_1() throws Exception { // file: project-docs/pages/migrating-sdk-code-to-3.n.adoc line: 23
+  public void migrating_sdk_code_to_3_n_1() throws Exception { // file:
+                                                               // project-docs/pages/migrating-sdk-code-to-3.n.adoc
+                                                               // line: 23
```

Comment:
this formatting is a bit odd.
Also is it just a manual denormalized comment (like we agreed we would remove from the Python examples recently) ?

Reply:
Ah yeah I can probably get rid of it, VSCode kind of forced this formatting but the comments aren't really necessary indeed.